### PR TITLE
Feature/data hub 10 15

### PR DESCRIPTION
**Pull request by cheinz-sag**

### CODE REVIEW

**Comment by cheinz-sag** [comment]
Next Tim will review the stuff and provide feedback; then doc colleagues can do a final review.

**Comment by cheinz-sag** [comment]
Hi Beate and Marlene,

can you please review the PR and:
- merge the contents to 10.15
- merge the contents to 10.13 and 10.14 once Michael or Tim give you a headsup (requires a fix to be built and rolled out)

Thanks
Christoph

File: content/datahub/operating-datahub-bundle/viewing-audit-logs.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ For each ad-hoc query, the following information is provided:
 
 | Column name | Description
 | ---         |  ---
+| User | The user name of the Dremio user, which has been used to execute the query
```
Comment (manasesjesus) [inline content/datahub/operating-datahub-bundle/viewing-audit-logs.md:33]: it is used as one word [in the Dremio docs](https://docs.dremio.com/software/security/rbac/users/#adding-users)
```suggestion
| User | The username of the Dremio user, which has been used to execute the query
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+In the navigator, select **Dremio users** under **Settings** to get an overview of all Dremio users created by an administrator of your {{< product-c8y-iot >}} DataHub tenant.
+
+The list of Dremio users with their corresponding properties is displayed. The context menu of each user provides actions to edit or delete a user.
+
+If the initial configuration has not been completed yet, no users are shown. If the initial configuration has been completed, the list includes the Dremio API user configured in the initial configuration.
+
+### Properties of a Dremio user
+
+#### User name
+The user name is a mandatory setting. It must be a unique value, i.e., no other Dremio user has the same user name. It consists of the tenant id plus forward slash and a string with a minimum length of three, starting with a character, and consisting of numbers, characters, dash, or underline.
```
Comment (manasesjesus) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:25]: it is used as one word [in the Dremio docs](https://docs.dremio.com/software/security/rbac/users/#adding-users)
```suggestion
#### Username
The username is a mandatory setting. It must be a unique value, i.e., no other Dremio user has the same username. It consists of the tenant ID plus forward slash and a string with a minimum length of three, starting with a character, and consisting of numbers, characters, dash, or underline.
```

An example of a valid username would be good here.

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+#### Password
+The password must have at least 8 characters with at least one letter and one number. 
+
+### Adding a Dremio user
+To add a Dremio user, select **Dremio users** under **Settings** and click **Add user** at the right of the top menu bar. In the new user form, provide the corresponding Dremio user properties.
+
+Click **Save** to save the settings and create the new user. Click **Cancel** to cancel the creation of the user.
+
+### Editing a Dremio user
+The **Dremio user** section under **Settings** displays the list of users. For each user, there is a context menu on the right side. Select **Edit** from that menu to edit a user. Except for the user name, all settings can be changed. The password can also optionally be changed by clicking **Change password**. Click **Save** to apply the new settings.
```
Comment (manasesjesus) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:54]: it is used as one word [in the Dremio docs](https://docs.dremio.com/software/security/rbac/users/#adding-users)
```suggestion
The **Dremio user** section under **Settings** displays the list of users. For each user, there is a context menu on the right side. Select **Edit** from that menu to edit a user. Except for the username, all settings can be changed. The password can also optionally be changed by clicking **Change password**. Click **Save** to apply the new settings.

```

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 4]
```diff
@@ -104,13 +108,15 @@ For **Azure Storage**, **Amazon S3**, and **HDFS** data lakes, you can also defi
...
+* A source in Dremio is created which connects to the Operational Store of {{< product-c8y-iot >}}. That source is not visible to the Dremio API user.
 * A space in Dremio is created which you can use to organize your custom Dremio entities such as views. The name of the space is your tenant ID concatenated with 'Space', for example, t12345Space.
 
 ### Editing settings
-Editing the settings is not supported. You must delete the existing settings and define new settings. If you want to keep your offloading configurations, you must export the configurations to a backup file, delete the settings, define new settings, and import the configurations from the backup file. See the section [Importing/exporting offloading configurations](/datahub/working-with-datahub/#import-export) for details on import/export.
+To edit the Dremio API user, click **Edit** in the **Dremio API user** section of the **Initial configuration** page. An edit form is opened which allows you to edit all user details, except for the user name, which is fixed. In section [Setting up additional Dremio users](/datahub/setting-up-datahub/#setting-up-dremio-users), all user details are described.
```
Comment (manasesjesus) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:117]:
```suggestion
To edit the Dremio API user, click **Edit** in the **Dremio API user** section of the **Initial configuration** page. An edit form is opened which allows you to edit all user details, except for the username, which is fixed. In section [Setting up additional Dremio users](/datahub/setting-up-datahub/#setting-up-dremio-users), all user details are described.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md (hunk 2 above)
Comment (cheinz-sag) [reply content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md]: Done...

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md (hunk 3 above)
Comment (cheinz-sag) [reply content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md]: Done...

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md (hunk 4 above)
Comment (cheinz-sag) [reply content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md]: Done

File: content/datahub/operating-datahub-bundle/viewing-audit-logs.md (hunk 1 above)
Comment (cheinz-sag) [reply content/datahub/operating-datahub-bundle/viewing-audit-logs.md]: Done...

File: content/datahub/datahub-overview-bundle/documentation-overview.md [hunk 5]
```diff
@@ -16,4 +16,4 @@ For your convenience, here is an overview of the contents of this document:
 | [Operating {{< product-c8y-iot >}} DataHub](/datahub/operating-datahub) | Run administrative tasks |
 | [Running {{< product-c8y-iot >}} DataHub on {{< product-c8y-iot >}} Edge](/datahub/running-datahub-on-the-edge) | Run the Edge edition of {{< product-c8y-iot >}} DataHub |
 | [Integrating {{< product-c8y-iot >}} DataHub with other {{< company-sag >}} products](/datahub/integrating-datahub-with-other-products) | Learn how to integrate {{< product-c8y-iot >}} DataHub with other {{< company-sag >}} products |
-| [Release notes](/datahub/datahub-release-notes) | Get news about the latest {{< product-c8y-iot >}} DataHub releases |
+| [Release notes](https://cumulocity.com/releasenotes/about/introduction/) | Get news about the latest {{< product-c8y-iot >}} DataHub releases |
```
[the remark this answers is not in the document]
Comment (cheinz-sag) [reply content/datahub/datahub-overview-bundle/documentation-overview.md]: Done

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
+---
+weight: 40
+title: Setting up Dremio users
+layout: redirect
+---
+
+In the [initial configuration](/datahub/setting-up-datahub/#setting-up-initial-configuration) of {{< product-c8y-iot >}} DataHub, the Dremio API user is configured. This user is required for the proxy REST API, which allows you to interact with Dremio using {{< product-c8y-iot >}} DataHub. This user can also be used to directly interact with Dremio in applications, using JDBC, ODBC, or REST API.
+
+Some use-cases might require more than one Dremio user for the interaction with Dremio. For that purpose, additional Dremio users can be added.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:9]:
```suggestion
Some use cases might require more than one Dremio user for the interaction with Dremio. For that purpose, additional Dremio users can be added.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+In the navigator, select **Dremio users** under **Settings** to get an overview of all Dremio users created by an administrator of your {{< product-c8y-iot >}} DataHub tenant.
+
+The list of Dremio users with their corresponding properties is displayed. The context menu of each user provides actions to edit or delete a user.
+
+If the initial configuration has not been completed yet, no users are shown. If the initial configuration has been completed, the list includes the Dremio API user configured in the initial configuration.
+
+### Properties of a Dremio user
+
+#### Username
+The username is a mandatory setting. It must be a unique value, i.e., no other Dremio user has the same username. It consists of the tenant ID plus forward slash and a string with a minimum length of three, starting with a character, and consisting of numbers, characters, dash, or underline. For example, the username may be *t47110815/myUser*.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:25]:
```suggestion
The username is a mandatory setting. It must be a unique value, that is, no other Dremio user has the same username. It consists of the tenant ID plus forward slash and a string with a minimum length of three, starting with a character, and consisting of numbers, characters, dash, or underline. For example, the username may be *t47110815/myUser*.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+#### First name, last name and email
+The first name, last name, and email of a Dremio user are optional settings.
+
+#### Permissions for data lake and space
+During the initial configuration of {{< product-c8y-iot >}} DataHub, a so-called source in Dremio is created, which connects Dremio with the data lake. Additionally, a so-called space is created in Dremio, in which Dremio artifacts like views can be organized.
+
+The Dremio user can be assigned additional permissions for the data lake source and the space. If the user has the permission for the data lake source assigned, the user can manage grants on that source for other users as well. The same applies to the space permission. Data lake permission and space permission are independent of each other; the setting of one permission does not affect the setting of the other.
+
+Having the corresponding permission assigned, the user can grant other Dremio users, which do not necessarily relate to {{< product-c8y-iot >}} DataHub, different permissions on the data lake source or the space, e.g., for reading data from the data lake or creating a table in the data lake.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:35]:
```suggestion
Having the corresponding permission assigned, the user can grant other Dremio users, which do not necessarily relate to {{< product-c8y-iot >}} DataHub, different permissions on the data lake source or the space, for example, for reading data from the data lake or creating a table in the data lake.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+For example, IoT data has been offloaded to the data lake using {{< product-c8y-iot >}} DataHub. A data scientist from a different business unit now wants to access the data lake contents. A Dremio account needs to be created for the data scientist. Therefore, a Dremio user created by {{< product-c8y-iot >}} DataHub, having the data lake permission, grants read access on the data lake source to the Dremio account of the data scientist. 
+
+{{< c8y-admon-info >}}
+Dremio refers to the permissions as [privileges](https://docs.dremio.com/software/security/rbac/privileges/). Privileges include for example `Select`, `Alter Table`, `Create Table`, or `Drop table`. Given a Dremio user with the corresponding permissions, permissions to other users can be granted via the Dremio UI. In the UI, browse to the data lake or space and select **Edit details** in the action menu. In the edit form, the list of privileges for all users is shown, with the option to update privileges and users. Alternatively, you can use the [Dremio SQL API](https://docs.dremio.com/software/sql-reference/sql-commands/rbac/) to modify privileges.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:40]:
```suggestion
Dremio refers to the permissions as [privileges](https://docs.dremio.com/software/security/rbac/privileges/). Privileges include for example `Select`, `Alter Table`, `Create Table`, or `Drop table`. Given a Dremio user with the corresponding permissions, permissions to other users can be granted via the Dremio UI. In the UI, browse to the data lake or space and select **Edit details** in the context menu. In the edit form, the list of privileges for all users is shown, with the option to update privileges and users. Alternatively, you can use the [Dremio SQL API](https://docs.dremio.com/software/sql-reference/sql-commands/rbac/) to modify privileges.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+{{< /c8y-admon-info >}}
+
+For each user, including the Dremio API user, the `manage grants` permissions on data lake and space are initially not set.
+
+#### Password
+The password must have at least 8 characters with at least one letter and one number. 
+
+### Adding a Dremio user
+To add a Dremio user, select **Dremio users** under **Settings** and click **Add user** at the right of the top menu bar. In the new user form, provide the corresponding Dremio user properties.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-dremio-users.md:49]:
```suggestion
To add a Dremio user, select **Dremio users** under **Settings** and click **Add user** at the right of the top menu bar. In the editor, provide the corresponding Dremio user properties.
```

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 11]
```diff
@@ -1,34 +1,34 @@
 ---
 weight: 30
-title: Setting up Dremio account and data lake
+title: Setting up initial configuration
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:3]:
```suggestion
title: Setting up the initial configuration
```

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 12]
```diff
@@ -1,34 +1,34 @@
...
 
-The setup of {{< product-c8y-iot >}} DataHub requires you to select a Dremio account name, and provide credentials to the data lake. In the navigator, select **Dremio & Data Lake** under **Settings** to define those settings.
+The setup of {{< product-c8y-iot >}} DataHub requires you to configure a Dremio API user and access to a data lake. In the navigator, select **Initial configuration** under **Settings** to define those settings.
 
 {{< c8y-admon-info >}}
 You need administration permissions to define the settings. See the section on [Defining {{< product-c8y-iot >}} DataHub permissions and roles](/datahub/setting-up-datahub#defining-permissions) for details.
 {{< /c8y-admon-info >}}
 
-### Defining new settings
+### Defining initial configuration
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:13]:
```suggestion
### Defining the initial configuration
```

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 13]
```diff
@@ -1,34 +1,34 @@
...
-1. tenant id
-2. forward slash
-3. string with a minimum length of two, starting with a character, and consisting of numbers, characters, dash, or underline
+Therefore, you have to configure in the initial configuration under **Dremio API user** the name and the password of that Dremio API user.
 
-If your tenant id is `t12345`, then `t12345/user` is a valid name. The system would also set this value as the initial value in the account field.
+The name is composed of two parts, with the first part being fixed:
 
-The password of the Dremio account must have at least eight characters, including at least one character and one number.
+1. Tenant id plus forward slash
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:23]:
```suggestion
1. Tenant ID plus forward slash
```

File: content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md [hunk 14]
```diff
@@ -104,13 +108,15 @@ For **Azure Storage**, **Amazon S3**, and **HDFS** data lakes, you can also defi
...
+* A source in Dremio is created which connects to the Operational Store of {{< product-c8y-iot >}}. That source is not visible to the Dremio API user.
 * A space in Dremio is created which you can use to organize your custom Dremio entities such as views. The name of the space is your tenant ID concatenated with 'Space', for example, t12345Space.
 
 ### Editing settings
-Editing the settings is not supported. You must delete the existing settings and define new settings. If you want to keep your offloading configurations, you must export the configurations to a backup file, delete the settings, define new settings, and import the configurations from the backup file. See the section [Importing/exporting offloading configurations](/datahub/working-with-datahub/#import-export) for details on import/export.
+To edit the Dremio API user, click **Edit** in the **Dremio API user** section of the **Initial configuration** page. An edit form is opened which allows you to edit all user details, except for the username, which is fixed. In section [Setting up additional Dremio users](/datahub/setting-up-datahub/#setting-up-dremio-users), all user details are described.
```
Comment (BeateRixen) [inline content/datahub/setting-up-datahub-bundle/setting-up-initial-configuration.md:117]:
```suggestion
To edit the Dremio API user, click **Edit** in the **Dremio API user** section of the **Initial configuration** page. In the dirotr you can edit all user details, except for the username, which is fixed. In section [Setting up additional Dremio users](/datahub/setting-up-datahub/#setting-up-dremio-users), all user details are described.
```

File: content/datahub/working-with-datahub-bundle/datahub-best-practices.md [hunk 15]
```diff
@@ -68,7 +66,42 @@ src."creationTime"."date" > {ts '2020-02-08 14:00:00.000'}
...
-
-For a dedicated environment, the additional data can be located somewhere else, provided it can be accessed via Dremio, for example, in a relational database. For performance and cost reasons, however, data and processing should always be colocated.
+Main use-case of {{< product-c8y-iot >}} DataHub is to offload data from the internal {{< product-c8y-iot >}} database
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/datahub-best-practices.md:69]:
```suggestion
Main use case of {{< product-c8y-iot >}} DataHub is to offload data from the internal {{< product-c8y-iot >}} database 

```

File: content/datahub/working-with-datahub-bundle/datahub-best-practices.md [hunk 16]
```diff
@@ -68,7 +66,42 @@ src."creationTime"."date" > {ts '2020-02-08 14:00:00.000'}
...
+one or more columns. First step is to stop and delete the offloading configuration associated with this target table. 
+Next, you can either externally rewrite the data or use Dremio's [CTAS query](https://docs.dremio.com/software/sql-reference/sql-commands/tables/) 
+feature which queries the data from this target table, filters obsolete data, and uses projections to skip unwanted columns. 
+The query results must be written to a new folder of the data lake. To make the folder accessible as a table in Dremio, 
+promote the folder to a dataset. Then, you can delete the old folder from the data lake. Next, you have to define a new offloading 
+configuration which uses the new folder name as target table name. Define the same filter criteria and columns as in 
+the CTAS query to ensure that the data has the same format. Save the offloading configuration, select the append mode 
+for storing the data, and activate the offloading.
+
+It must be emphasized that this use case is not officially supported by {{< product-c8y-iot >}} DataHub. While there is
```
Comment (BeateRixen) [inline content/datahub/working-with-datahub-bundle/datahub-best-practices.md:102]: @cheinz-sag Should this better be a "caution" admonition?

**Review by BeateRixen — APPROVED** [review]

**Review by manasesjesus — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**